src/news/__tests__/newsActions.test.ts

```diff
@@ -1,37 +1,61 @@
+import { mockApiFailFetch } from '__mock__/api.fail.mock'
 import { mockApi } from '__mock__/api.mock'
 import { mockGlobalState } from '__mock__/globalstate.mock'
 import { generateMockStore } from '__mock__/mockStore.mock'
+import { mockNews } from '__mock__/news.mock'
 import {
   getNewsData,
+  getNewsFetchFailedAction,
+  NEWS_FETCH_FAILED,
   NEWS_RECEIVED_ACTION,
   NewsReceivedAction,
   receiveNewsAction
 } from 'news/newsActions'
 import { MockStore } from 'redux-mock-store'
 
 describe('actions', () => {
-  let store: MockStore<{}>
-  const newsItems = mockGlobalState.news
-  const expectedAction: NewsReceivedAction = {
-    type: NEWS_RECEIVED_ACTION,
-    newsItems
-  }
-
-  beforeEach(() => {
-    store = generateMockStore(mockGlobalState, mockApi)
-    store.clearActions()
-  })
 
   describe('synchronous actions', () => {
     it('receiveNewsAction should create a news received action', () => {
-      expect(receiveNewsAction(newsItems)).toEqual(expectedAction)
+      const newsItems = mockGlobalState.news
```

Comment:
I think it's better for the test code structure to be consistent across tests of different components?